pkg/cmd/root/help_topic.go

```diff
@@ -104,32 +104,32 @@ var HelpTopics = map[string]map[string]string{
 			- %[1]stimeago <time>%[1]s: renders a timestamp as relative to now
 			- %[1]stimefmt <format> <time>%[1]s: formats a timestamp using Go's Time.Format function
 			- %[1]struncate <length> <input>%[1]s: ensures input fits within length
-		`, "`"),
```

Comment:
This argument to Docf was necessary to insert a literal `` ` `` into a backtick-delimited string. Please don't remove it.

Reply:
Done.

A minor change I made is remove trailing `\` since the shell can already handle multi-line input when users press enter after an opening quote.